notebook/tests/selenium/test_dashboard_nav.py

```diff
@@ -13,32 +13,43 @@ def get_list_items(browser):
     } for a in browser.find_elements_by_class_name('item_link')]
 
 
+def only_dir_links(browser):
+    try:
+        assert 'tree' in browser.current_url
+    except AssertionError:
+        raise("You currently ")
+    wait_for_selector(browser, '.item_link')
+    items = get_list_items(browser)
+    return [i for i in items if 'tree' in i['link'] and i['label'] != '..']
```

Comment:
This will go wrong for any file that has `tree` in the name. Maybe that's unlikely, but I think it's worth doing it a bit more thoroughly.

Reply:
ok addressed by using an explicit check for whether urls point to something in the tree (with  `url_in_tree`)